extensions/amp-autocomplete/amp-autocomplete.md

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-autocomplete`
+
+**This feature is experimental and activated by the `amp-autocomplete` experiment.**
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>An autocomplete-enabled input field suggests completed results corresponding to the user input as they type into the input field.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Experimental</td>
```

Comment:
Let's link to the experiment docs like some other extension `README.md`s do
https://github.com/ampproject/amphtml/blob/71a625ee9ff53706314cc10632c39b864e23c34b/extensions/amp-script/amp-script.md#L35